test/functional/services/combo_box.js

```diff
@@ -55,6 +56,7 @@ export function ComboBoxProvider({ getService }) {
 
     async _waitForOptionsListLoading(comboBoxElement) {
       await comboBoxElement.waitForDeletedByClassName('euiLoadingSpinner');
+      await PageObjects.common.sleep(1000);
```

Comment:
Is there any element we could wait for once this is done loading, so we wouldn't need to wait with an arbitrary amount of time here?

Reply:
Unfortunately not, the problem here is that right before rendering filtered list, there's a flicker of unfiltered list with the same structure. We could try to refactor a comboBox or think about refactoring the whole test service, but as it's just 6.8, I feel like it's not worth the effort, especially because the solution above works well.